AUTHORS

```diff
@@ -21,3 +21,4 @@ Matteo Pasotti <matteo@mageia.it>
 @kulti <kultihell@gmail.com>
 @pisculichi <pisculichi@openmailbox.org>
 Sérgio Marques <smarquespt@gmail.com>
+Valter Nazianzeno <manipuladordedados@gmail.com>
```

Comment:
I'd say one has to contribute code in order to make it to the AUTHORS list, but the two previous guys have also contributed only translations. @agaida, @jleclanche what's the policy here?